Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/categories-tree/CategoriesTree.tsx

```diff
@@ -0,0 +1,105 @@
+﻿import React, { useCallback, useEffect, useState } from 'react';
+import { TreeItem, TreeView } from '@material-ui/lab';
+import { isArray, isEmpty, without } from 'lodash';
+import MdExpandMoreIcon from '../../guidelines/icons/MdExpandMoreIcon';
+import MdChevronRightIcon from '../../guidelines/icons/MdChervronRightIcon';
+import ITreeItemType from '../../../common/tree-types';
+import { ITreeProps } from '../../guidelines/trees/Tree';
+
+import styles from './CategoriesTree.module.scss';
+
+interface ICategoriesTreeProps extends ITreeProps{
+}
+
+const CategoryTree = ({
+    items,
+    onCategoryLabelClick,
+    defaultSelected,
+    defaultExpanded = [],
+}: ICategoriesTreeProps) => {
+    const [ expandedCategoriesIds, setExpanded ] = useState([] as string[]);
+    const [ selectedCategoriesIds, setSelected ] = useState('');
```

Comment:
Also this is named `selectedCategoriesIds`, i.e. plural, but holds a single string. Is this correct?